### PR TITLE
Añadir check regualtorio

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

- Se añade la documentacion del check regulatior

### Tasks:

- [x] Añadir documentacion del check regulario
- [x] Traducir documentacion

### Links:

> Add links for each file modified

* [CA](http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/ca/distri/adm-pub/cnmc/#check-regulatori)
* [ES](http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/es/distri/adm-pub/cnmc/#check-regulatori)

### CODE REVIEW

**Comment by jaumef** [comment]
El test falla por: https://github.com/gisce/powerp-docs/issues/342

Pendiente de solucionar.
No hacer "merge" ya que no se hará "build" de la documentación

**Comment by jaumef** [comment]
El test falla por: https://github.com/gisce/powerp-docs/issues/342

Pendiente de solucionar.
No hacer "merge" ya que no se hará "build" de la documentación

**Comment by jaumef** [comment]
El test falla por: https://github.com/gisce/powerp-docs/issues/342
No hacer MERGE ya que no se puede hacer BUILD

**Comment by jaumef** [comment]
El test falla por: https://github.com/gisce/powerp-docs/issues/342
No hacer MERGE ya que no se puede hacer BUILD

**Comment by jaumef** [comment]
El test falla por: https://github.com/gisce/powerp-docs/issues/342
No hacer MERGE ya que no se puede hacer BUILD

File: docs/distri/adm-pub/cnmc.md [hunk 1]
```diff
@@ -179,3 +179,11 @@ L'assistent obre una finestra amb el botó **Exportar**. Prement el botó es gen
 Un cop a la darrera finestra, amb el botó **Obrir** s'accedeix al contingut del **ZIP** generat.
 
 ![](../_static/cnmc/sips_4.png)
+
+## Check regulatori
+
+- Als models involurats en els informes d'administracio publica existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest te tres opcions:
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]:
```suggestion
- Als models involucrats en els informes d'administració pública, existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest camp té tres opcions:
```

Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]:
```suggestion
- Als models involucrats en els informes d'administració pública, existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest camp té tres opcions:
```

File: docs/distri/adm-pub/cnmc.md [hunk 2]
```diff
@@ -179,3 +179,11 @@ L'assistent obre una finestra amb el botó **Exportar**. Prement el botó es gen
...
+
+   - **Segons criteri**: Se usa el criterio descrito en el `BOE`. Es el valor por defecto y se puede consultar en fichero de la resolucion implicada.
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: - Descripció en castellà
- S'esmena un BOE, però no hi ha enllaç a aquest

File: docs/distri/adm-pub/cnmc.md (hunk 1 above)
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]:
```suggestion
- Als models involucrats en els informes d'administració pública, existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest camp té tres opcions:
```

Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]:
```suggestion
- Als models involucrats en els informes d'administració pública, existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest camp té tres opcions:
```

File: docs/distri/adm-pub/cnmc.md (hunk 2 above)
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: - Descripció en castellà
- S'esmena un BOE però no hi ha referència a aquest

**Comment by jaumef** [comment]
Fer Merge amb `master` abans de continuar

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/adm-pub/cnmc.md [hunk 3]
```diff
@@ -179,3 +179,11 @@ L'assistent obre una finestra amb el botó **Exportar**. Prement el botó es gen
 Un cop a la darrera finestra, amb el botó **Obrir** s'accedeix al contingut del **ZIP** generat.
 
 ![](../_static/cnmc/sips_4.png)
+
+## Check regulatori
+
+- Als models involucrats en els informes d'administracio pública existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest te tres opcions:
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: S'inicia com a llistat però es un paràgraf:

```suggestion
Als models involucrats en els informes d'administracio pública existeix el camp **Criteri regulatori**. Aquest camp ens permet personalitzar els elements que apareixen als informes. Aquest te tres opcions:
```

File: docs/distri/adm-pub/cnmc.md [hunk 4]
```diff
@@ -179,3 +179,11 @@ L'assistent obre una finestra amb el botó **Exportar**. Prement el botó es gen
...
+
+   - **Segons criteri**: Es fa servir el criteri descrit al [BOE](https://www.boe.es/boe/dias/2017/04/28/pdfs/BOE-A-2017-4666.pdf). Es el valor per defecte i es pot consutlar en el fitxer de la resolució implicada.
```
Comment (jaumef) [inline docs/distri/adm-pub/cnmc.md]: No es tradueix, s'ha de revisar la traducció

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_check_regulatori/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**